repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/dataset.ftlh

```diff
@@ -1,11 +1,14 @@
-<#include "../include/imports.ftl">
-<#include "./macro/structured-text.ftl">
+<#include "../include/imports.ftlh">
+<#include "./macro/structured-text.ftlh">
 <@hst.setBundle basename="publicationsystem.headers"/>
 
 <#assign dateFormat="dd/MM/yyyy"/>
 <#assign formatFileSize="uk.nhs.digital.ps.directives.FileSizeFormatterDirective"?new() >
 <#assign formatCoverageDates="uk.nhs.digital.ps.directives.CoverageDatesFormatterDirective"?new() >
 
+<p>Output format: ${.output_format}
+<p>Auto-escaping: ${.auto_esc?c}
+
```

Comment:
Is this just for testing?

Reply:
yes, forgot to remove before commit :-).  Will do that now...